### PR TITLE
[Feature] [Beta] Implement Additional Grunt Dialogue

### PR DESCRIPTION
**Pull request by Blitz425**
## What are the changes?
Added additional dialogue options for Evil Team Grunts

## Why am I doing these changes?
Per request of Temp, this PR aims to add in more personality for the Grunt members. It is currently awkward right now with the amount of grunts you can find in a run all saying the exact same thing, so this PR addresses that by adding in more lines and references from the mainline titles that players might enjoy seeing.

## What did change?
Added 4 Additional Encounter / Victory lines for Rocket/Magma/Aqua/Galactic/Plasma/Flare

## How to test the changes?
Get to a wave that spawns in a Grunt

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/en/dialogue.ts [hunk 1]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "magma_grunt": {
     "encounter": {
-      1: " If you get in the way of Team Magma, don’t expect any mercy!"
+      1: "If you get in the way of Team Magma, don’t expect any mercy!",
+      2: "You'd better not interfere with our plans! We're making the world a better place!",
+      3: "You're in the way! Team Magma has no time for kids like you!",
+      4: "I hope you brought marshmallows because things are about to heat up!",
+      5: "We're going to use the power of a volcano! It's gonna be.. explosive! Get it? Heh heh!"
```
Comment (Tempo-anon) [inline src/locales/en/dialogue.ts:408]: nit: is this supposed to be 3 periods rather than just 2?

File: src/locales/en/dialogue.ts [hunk 2]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Huh? I lost?!"
+      1: "Huh? I lost?!",
+      2: "I can't believe I lost! I even skipped lunch for this",
+      3: "No way! You're just a kid!",
+      4: "Urrrgh...I should've ducked into our hideout right away...",
```
Comment (Tempo-anon) [inline src/locales/en/dialogue.ts:414]: nit: space after `Urrrgh...`

File: src/locales/en/dialogue.ts [hunk 3]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      5: "You beat me... Do you think the boss will dock my pay for this?"
     },
   },
   "aqua_grunt": {
     "encounter": {
-      1: "No one who crosses Team Aqua gets any mercy, not even kids!"
+      1: "No one who crosses Team Aqua gets any mercy, not even kids!",
+      2: "Grrr…You've got some nerve meddling with Team Aqua!",
```
Comment (Tempo-anon) [inline src/locales/en/dialogue.ts:421]: Is this the wrong `...`?

File: src/locales/en/dialogue.ts [hunk 4]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      3: "You're about to get soaked! And not just from my water Pokémon!",
+      4: "We, Team Aqua, exist for the good of all!",
+      5: "Prepare to be washed away by the tides of my… uh, Pokémon! Yeah, my Pokémon!"
     },
     "victory": {
-      1: "You're kidding me!"
+      1: "You're kidding me!",
+      2: "Arrgh, I didn't count on being meddled with by some meddling kid!",
+      3: "I lost?! Guess I'll have to swim back to the hideout now...",
+      4: "Oh, man, what a disaster… The boss is going to be furious…",
```
Comment (Tempo-anon) [inline src/locales/en/dialogue.ts:430]: Same thing about the `...` here that appears twice

File: src/locales/en/dialogue.ts [hunk 5]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "How?!",
+      5: "Note to self: practice Pokémon battling, ASAP."
     },
   },
   "plasma_grunt": {
     "encounter": {
-      1: "We won't tolerate people who have different ideas!"
+      1: "We won't tolerate people who have different ideas!",
+      2: "If I win against you, release your Pokémon!",
+      3: "Your Pokémon will serve a greater purpose under Lord N's vision.",
```
Comment (Tempo-anon) [inline src/locales/en/dialogue.ts:454]: Should we allude to `N` if he's not part of Team Plasma right now?

**Review by damocleas — APPROVED** [review]

File: src/locales/ko/dialogue.ts [hunk 6]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "We, Team Aqua, exist for the good of all!",
+      5: "Prepare to be washed away by the tides of my... uh, Pokémon! Yeah, my Pokémon!"
     },
     "victory": {
-      1: "말도 안돼!"
+      1: "말도 안돼!",
+      2: "Arrgh, I didn't count on being meddled with by some meddling kid!",
+      3: "I lost?! Guess I'll have to swim back to the hideout now...",
+      4: "Oh, man, what a disaster... The boss is going to be furious...",
+      5: "You beat me... Do you think the boss will make me walk the plank for this?"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:431]:
```suggestion
      2: "크윽, 참견쟁이 꼬마에게 당하다니!",
      3: "내가 졌다고?! 헤엄쳐서 은신처로 돌아가야겠군..",
      4: "이런, 완전 망했군... 보스가 화를 내실텐데..",
      5: "네가 이겼어... 이것 때문에 보스가, 나를 판자 위로 보내는 거 아냐?"
```

File: src/locales/ko/dialogue.ts [hunk 7]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "galactic_grunt": {
     "encounter": {
-      1: "갤럭시단을 방해하지 마!"
+      1: "갤럭시단을 방해하지 마!",
+      2: "Witness the power of our technology and the future we envision!",
+      3: "In the name of Team Galactic, I'll eliminate anyone who stands in our way!",
+      4: "Get ready to lose!",
+      5: "Hope you're ready for a cosmic beatdown!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:440]:
```suggestion
      2: "기술의 힘과 우리가 꿈꾸는 미래를 목격하라!",
      3: "갤럭시단으로서, 우리를 방해하는 자는 누구든 제거하겠다!",
      4: "질 준비나 하라고!",
      5: "우주에서 싸울 준비는 됐겠지!"
```

File: src/locales/ko/dialogue.ts [hunk 8]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "사격 중지…… "
+      1: "사격 중지…… ",
+      2: "This setback means nothing in the grand scheme.",
+      3: "Our plans are bigger than this defeat.",
+      4: "How?!",
+      5: "Note to self: practice Pokémon battling, ASAP."
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:447]:
```suggestion
      2: "이런 좌절이라도, 우리의 큰 계획 앞엔 아무 의미도 안돼",
      3: "우리의 계획은 이번 패배보다 크거든.",
      4: "어떻게 한 거지?!",
      5: "메모해야겠군, 최대한 포켓몬 배틀을 ASAP으로 연습할 것."
```

File: src/locales/ko/dialogue.ts [hunk 9]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "plasma_grunt": {
     "encounter": {
-      1: "다른 생각을 가진사람들은 용납하지 않겠다!"
+      1: "다른 생각을 가진사람들은 용납하지 않겠다!",
+      2: "If I win against you, release your Pokémon!",
+      3: "If you get in the way of Team Plasma, I'll take care of you!",
+      4: "Team Plasma will liberate Pokémon from selfish humans like you!",
+      5: "Our hairstyles are out of this world... but our battling skills? You'll find out soon enough."
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:456]:
```suggestion
      2: "내가 이기면, 네 포켓몬들을 놓아주도록 해!",
      3: "플라즈마단을 방해한다면, 내가 처리해주지!",
      4: "플라즈마단은 너 같은 이기적인 인간에게서 포켓몬을 해방시킬 것이다!",
      5: "우리 스타일링은 정말 대단하지... 배틀 실력은어떻냐고? 곧 알게 될거야."
```

File: src/locales/ko/dialogue.ts [hunk 10]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "플라-스마-!"
+      1: "플라-스마-!",
+      2: "How could I lose...",
+      3: "...What a weak Pokémon, I'll just have to go steal some better ones!",
+      4: "Great plans are always interrupted.",
+      5: "This is bad... Badbadbadbadbadbadbad! Bad for Team Plasma! Or Plasbad, for short!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:463]:
```suggestion
      2: "내가 지다니...",
      3: "...포켓몬이 너무 약해, 더 좋은 포켓몬을 훔치러 가야겠군!",
      4: "훌륭한 계획은 항상 방해를 받는다니깐.",
      5: "이건 나빠... 나빠나빠나빠나빠나빠! 플라즈단에 나빠! 줄여서, 플라나빠!"
```

File: src/locales/ko/dialogue.ts [hunk 11]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "flare_grunt": {
     "encounter": {
-      1: "패션이 우리한텐 가장 중요하다고!"
+      1: "Your Pokémon are no match for the elegance of Team Flare.",
+      2: "Hope you brought your sunglasses, because things are about to get bright!",
+      3: "Team Flare will cleanse the world of imperfection!",
+      4: "Prepare to face the brilliance of Team Flare!",
+      5: "Fashion is most important to us!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:472]:
```suggestion
      1: "네 포켓몬, 플레어단의 우아함에 상대가 되지 않는다고.",
      2: "선글라스를 가져왔길 바랄게, 곧 밝게 빛날테니깐!",
      3: "플레어단이 세상의 불완전함을 정화할거야!",
      4: "플레어단의 광채를 마주칠 준비는 됐겠지!",
      5: "패션이 우리한텐 가장 중요하다고!"
```

File: src/locales/ko/dialogue.ts [hunk 12]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "미래가 밝아 보이질 않네."
+      1: "The future doesn't look bright for me.",
+      2: "Perhaps there's more to battling than I thought. Back to the drawing board.",
+      3: "Gahh?! I lost?!",
+      4: "Even in defeat, Team Flare's elegance shines through.",
+      5: "You may have beaten me, but when I lose, I go out in style!"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:479]:
```suggestion
      1: "미래가 밝아 보이질 않네.",
      2: "생각했던 거 보다 전투에는 많은 요소가 있는 것 같군. 다시 계획을 짜야겠어.",
      3: "아앗?! 내가 졌어?!",
      4: "패배 속에서도, 플레어단의 우아함은 빛나고 있다고.",
      5: "네가 이긴 것 같네, 그치만 내가 졌어도, 난 우아하게 나갈거니까!"
```

**Review by sodaMelon — COMMENTED** [review]
added korean localization

File: src/locales/de/dialogue.ts [hunk 13]
```diff
@@ -372,10 +372,18 @@ export const PGMdialogue: DialogueTranslationEntries = {
       1: `Jetzt gibt es Ärger!…
           $und es kommt noch härter!
           $Wir wollen über die Erde regieren…
-          $und naja du kennst den Rest…!`
+          $und naja du kennst den Rest…!`,
+      2: "We're pulling a big job here! Get lost, kid!",
+      3: "Hand over your Pokémon, or face the wrath of Team Rocket!",
+      4: "You're about to experience the true terror of Team Rocket!",
+      5: "Hey, kid! Me am a Team Rocket member kind of guy!" //Use of wrong grammar is deliberate
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:379]:
```suggestion
      2: "Wir führen gerade eine große Operation durch. Hast du vor uns zu stören?",
      3: "Gib uns deine Pokémon, oder stelle dich dem Zorn von Team Rocket!"
      4: "Team Rocket wird seinen Plan zur Vollendung bringen, Aus dem Weg!",
      5: "Los, gib uns deine Pokémon. Wir brauchen sie für unseren Plan!"
```

File: src/locales/de/dialogue.ts [hunk 14]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "magma_grunt": {
     "encounter": {
-      1: "Keiner, der sich Team Magma in den Weg stellt, bekommt Gnade, nicht einmal Kinder!"
+      1: "Keiner, der sich Team Magma in den Weg stellt, bekommt Gnade, nicht einmal Kinder!",
+      2: "You'd better not interfere with our plans! We're making the world a better place!",
+      3: "You're in the way! Team Magma has no time for kids like you!",
+      4: "I hope you brought marshmallows because things are about to heat up!",
+      5: "We're going to use the power of a volcano! It's gonna be... explosive! Get it? Heh heh!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:419]:
```suggestion
      1: "Keiner, der sich Team Magma in den Weg stellt, bekommt Gnade, nicht einmal Kinder!",
      2: "Störe besser nicht unsere Pläne! Wir formen die Welt nach unseren Vorstellungen!",
      3: "Du stehst uns im Weg! Team Magma hat keine Zeit für Störenfriede wie dich!",
      4: "Bereite dich auf die Hölle vor, denn es wird bald sehr heiß!",
      5: "Wir werden die Macht des Vulkans entfesseln! Es wird gewaltig sein! Mach dich bereit!"
```

File: src/locales/de/dialogue.ts [hunk 15]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Wie kann das sein? Ich bin Teil vom mächtigen Team Magma! Wir wollen doch nur die Welt verbessern…"
+      1: "Wie kann das sein? Ich bin Teil vom mächtigen Team Magma! Wir wollen doch nur die Welt verbessern…",
+      2: "I can't believe I lost! I even skipped lunch for this",
+      3: "No way! You're just a kid!",
+      4: "Urrrgh... I should've ducked into our hideout right away...",
+      5: "You beat me... Do you think the boss will dock my pay for this?"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:426]:
```suggestion
 1: "Wie kann das sein? Ich bin Teil des mächtigen Team Magma! Wir streben nach der Verbesserung der Welt...",
      2: "Unglaublich, dass ich verloren habe! Mit meinen mächtigen Pokémon.",
      3: "Das kann nicht sein! Ich hab doch viel mehr Erfahrung als du!",
      4: "Verdammt... Ich hätte sofort in unser Versteck fliehen sollen...",
      5: "Du hast mich besiegt... Der Boss wird mich dafür zur Rechenschaft ziehen."
```

File: src/locales/de/dialogue.ts [hunk 16]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "aqua_grunt": {
     "encounter": {
       1: "Du willst dich also mit Team Aqua anlegen? Du traust dich ja was… Dich werfe ich über Bord!",
+      2: "Grrr... You've got some nerve meddling with Team Aqua!",
+      3: "You're about to get soaked! And not just from my water Pokémon!",
+      4: "We, Team Aqua, exist for the good of all!",
+      5: "Prepare to be washed away by the tides of my... uh, Pokémon! Yeah, my Pokémon!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:435]:
```suggestion
      2: "Du hast ganz schön Mut, dich mit Team Aqua anzulegen!",
      3: "Ich hoffe du hast einen Regenschirm dabei. Hier wird es jetzt nass!",
      4: "Wir, Team Aqua, existieren zum Wohle aller!",
      5: "Bereite dich darauf vor, von den Fluten meiner Pokémon weggespült zu werden!"
```

File: src/locales/de/dialogue.ts [hunk 17]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
       1: "Vielleicht sollte ich wohl lieber selber über die Planke gehen…",
+      2: "Arrgh, I didn't count on being meddled with by some meddling kid!",
+      3: "I lost?! Guess I'll have to swim back to the hideout now...",
+      4: "Oh, man, what a disaster... The boss is going to be furious...",
+      5: "You beat me... Do you think the boss will make me walk the plank for this?"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:442]:
```suggestion
      2: "Arrgh, ich habe nicht damit gerechnet, von einer Landratte gestört zu werden!",
      3: "Ich habe verloren?! Ich schätze, ich muss jetzt zurück zum Versteck schwimmen...",
      4: "Oh Mann, was für eine Katastrophe... Der Boss wird wütend sein...",
      5: "Du hast mich besiegt... Meinst du, der Boss wird mich dafür kielholen lassen?"
```

File: src/locales/de/dialogue.ts [hunk 18]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "galactic_grunt": {
     "encounter": {
-      1: "Team Galaktik wird die Welt in eine bessere Welt verwandeln! Und du wirst uns nicht aufhalten!"
+      1: "Team Galaktik wird die Welt in eine bessere Welt verwandeln! Und du wirst uns nicht aufhalten!",
+      2: "Witness the power of our technology and the future we envision!",
+      3: "In the name of Team Galactic, I'll eliminate anyone who stands in our way!",
+      4: "Get ready to lose!",
+      5: "Hope you're ready for a cosmic beatdown!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:451]:
```suggestion
      1: "Team Galaktik wird die Welt in eine bessere verwandeln! Und du wirst uns nicht aufhalten!",
      2: "Erlebe die Macht unserer Technologie und die Zukunft, die wir uns vorstellen!",
      3: "Im Namen von Team Galaktik werde ich jeden beseitigen, der uns im Weg steht!",
      4: "Mach dich bereit zu verlieren!",
      5: "Hoffentlich bist du bereit für eine kosmische Niederlage!"
```

File: src/locales/de/dialogue.ts [hunk 19]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Zyrus wird uns für diese Niederlage bestrafen…"
+      1: "Zyrus wird uns für diese Niederlage bestrafen…",
+      2: "This setback means nothing in the grand scheme.",
+      3: "Our plans are bigger than this defeat.",
+      4: "How?!",
+      5: "Note to self: practice Pokémon battling, ASAP."
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:458]:
```suggestion
      1: "Zyrus wird uns für diese Niederlage bestrafen…",
      2: "Dieser Rückschlag bedeutet nichts in Hinsicht unseres großen Plans.",
      3: "Unsere Pläne sind größer als diese Niederlage.",
      4: "Wie ist das möglich?!",
      5: "Notiz an mich selbst: Pokémon-Kämpfe üben, so bald wie möglich."
```

File: src/locales/de/dialogue.ts [hunk 20]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "plasma_grunt": {
     "encounter": {
-      1: "Pokémon sollten frei sein! Team Plasma wird sie befreien!"
+      1: "Pokémon sollten frei sein! Team Plasma wird sie befreien!",
+      2: "If I win against you, release your Pokémon!",
+      3: "If you get in the way of Team Plasma, I'll take care of you!",
+      4: "Team Plasma will liberate Pokémon from selfish humans like you!",
+      5: "Our hairstyles are out of this world... but our battling skills? You'll find out soon enough."
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:467]:
```suggestion
"suggestion": {
      1: "Pokémon sollten frei sein! Team Plasma wird sie befreien!",
      2: "Wenn ich gegen dich gewinne, lass deine Pokémon frei!",
      3: "Wenn du Team Plasma im Weg stehst, werde ich mich um dich kümmern!",
      4: "Team Plasma wird Pokémon von egoistischen Menschen wie dir befreien!",
      5: "Lass dich von unserem Aussehen nicht täuschen. Unsere Kampffähigkeiten sind überragend!"
    },

```

File: src/locales/de/dialogue.ts [hunk 21]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Wie konnte ich verlieren? Ich dachte, ich würde die Welt retten…"
+      1: "Wie konnte ich verlieren? Ich dachte, ich würde die Welt retten…",
+      2: "How could I lose...",
+      3: "...What a weak Pokémon, I'll just have to go steal some better ones!",
+      4: "Great plans are always interrupted.",
+      5: "This is bad... Badbadbadbadbadbadbad! Bad for Team Plasma! Or Plasbad, for short!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:474]:
```suggestion
      1: "Wie konnte ich verlieren? Ich dachte, ich würde die Welt retten...",
      2: "Wie konnte ich nur verlieren...",
      3: "...Dieses Pokémon ist zu schwach, ich werde stärkere beschaffen müssen!",
      4: "Große Pläne stoßen immer auf Hindernisse.",
      5: "Das ist ein schwerer Rückschlag für Team Plasma..."
```

File: src/locales/de/dialogue.ts [hunk 22]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
   },
   "flare_grunt": {
     "encounter": {
-      1: `Ich bin ein Mitglied von Team Flare! Das sieht man mir doch an. Mein Stil ist unverkennbar!
-          $Du kannst definitiv ein Umstyling gebrauchen!`
+      1: "Your Pokémon are no match for the elegance of Team Flare.",
+      2: "Hope you brought your sunglasses, because things are about to get bright!",
+      3: "Team Flare will cleanse the world of imperfection!",
+      4: "Prepare to face the brilliance of Team Flare!",
+      5: "Fashion is most important to us!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:483]:
```suggestion
      1: "Deine Pokémon haben keine Chance gegen die Überlegenheit von Team Flare.",
      2: "Mach dich bereit, denn gleich wird es hier lichterloh brennen!",
      3: "Team Flare wird die Welt von allen Makeln befreien!",
      4: "Bereite dich auf die unvergleichliche Macht von Team Flare vor!",
      5: "Unsere Mission steht über allem, sogar über der Mode!"
```

File: src/locales/de/dialogue.ts [hunk 23]
```diff
@@ -404,43 +412,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "Stil ist wohl doch nicht alles…"
+      1: "The future doesn't look bright for me.",
+      2: "Perhaps there's more to battling than I thought. Back to the drawing board.",
+      3: "Gahh?! I lost?!",
+      4: "Even in defeat, Team Flare's elegance shines through.",
+      5: "You may have beaten me, but when I lose, I go out in style!"
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:490]:
```suggestion
   1: "Diese Niederlage wirft einen Schatten auf meine Zukunft.",
      2: "Es scheint, dass ich meine Strategien überdenken muss. Zurück ans Reißbrett.",
      3: "Unglaublich?! Ich habe verloren?!",
      4: "Selbst in der Niederlage bleibt Team Flare unübertroffen in seiner Eleganz.",
      5: "Du hast mich besiegt, aber Team Flare wird immer in Glanz und Stil erstrahlen."
```

File: src/locales/zh_CN/dialogue.ts [hunk 24]
```diff
@@ -384,50 +384,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "flare_grunt": {
     "encounter": {
-      1: "时尚最重要！"
+      1: "Your Pokémon are no match for the elegance of Team Flare.",
+      2: "Hope you brought your sunglasses, because things are about to get bright!",
+      3: "Team Flare will cleanse the world of imperfection!",
+      4: "Prepare to face the brilliance of Team Flare!",
+      5: "Fashion is most important to us!"
```
Comment (40chyan) [inline src/locales/zh_CN/dialogue.ts:471]:
```suggestion
      1: "你的宝可梦无法与闪焰队的优雅相提并论",
      2: "带个墨镜吧，别被我闪瞎狗眼了！",
      3: "闪焰队将净化这个不完美的世界！",
      4: "准备面对闪焰队的美！",
      5: "时尚对我们来说最重要！"
```

File: src/locales/zh_CN/dialogue.ts [hunk 25]
```diff
@@ -384,50 +384,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
     "victory": {
-      1: "未来一片黑暗啊…"
+      1: "The future doesn't look bright for me.",
+      2: "Perhaps there's more to battling than I thought. Back to the drawing board.",
+      3: "Gahh?! I lost?!",
+      4: "Even in defeat, Team Flare's elegance shines through.",
+      5: "You may have beaten me, but when I lose, I go out in style!"
```
Comment (40chyan) [inline src/locales/zh_CN/dialogue.ts:478]:
```suggestion
      1: "我的未来看起来并不明亮…",
      2: "这战斗比我想的更难搞，我得重头训练了。",
      3: "啊啊？我输了？！",
      4: "就算是在失败当中，闪焰队依旧优雅动人！",
      5: "你虽然打败了我，但是我输的也这么潇洒！"
```

**Review by 40chyan — COMMENTED** [review]
zh_cn translation

File: src/locales/ko/dialogue.ts [hunk 26]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "으윽… 당장 은신처로 숨을 걸 그랬나…",
+      5: "네가 이겼어… 이것 때문에 보스가, 내 월급 깎으려나?"
     },
   },
   "aqua_grunt": {
     "encounter": {
-      1: "아쿠아단을 넘본 사람에게는 자비는 없다, 꼬마도 마찬가지야!"
+      1: "아쿠아단을 넘본 사람에게는 자비는 없다, 꼬마도 마찬가지야!",
+      2: "쯧… 아쿠아단에 참견하다니 오지랖이 넓군!",
+      3: "흠뻑 물을 뒤집어쓰게 될 꺼다! 내 물 포켓몬의 공격 뿐만이 아니야!",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:422]:
```suggestion
      3: "흠뻑 물을 뒤집어쓰게 될 거다! 내 물 포켓몬의 공격 뿐만이 아니야!",
```

File: src/locales/ko/dialogue.ts [hunk 27]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "우리, 아쿠아단은, 모두를 위해 존재한다!",
+      5: "내가 떠밀… 아니, 파도에 떠내려갈 준비나 하라고! 내 포켓몬이 그렇게 만들 테니까 "
     },
     "victory": {
-      1: "말도 안돼!"
+      1: "말도 안돼!",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:427]:
```suggestion
      1: "말도 안 돼!",
```

File: src/locales/ko/dialogue.ts [hunk 28]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      4: "질 준비나 하라고!",
+      5: "우주에서 싸울 준비는 됐겠지!"
     },
     "victory": {
-      1: "사격 중지…… "
+      1: "사격 중지…… ",
+      2: "이런 좌절이라도, 우리의 큰 계획 앞엔 아무 의미도 못 돼.",
+      3: "우리의 계획은 이번 패배보다 크거든.",
+      4: "어떻게 한 거지?!",
+      5: "메모해야겠군, 최대한 포켓몬 배틀을 ASAP으로 연습할 것."
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:447]:
```suggestion
      5: "메모해야겠군. 최대한 포켓몬 배틀을 ASAP으로 연습할 것."
```

File: src/locales/ko/dialogue.ts [hunk 29]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      3: "…포켓몬이 너무 약해, 더 좋은 포켓몬을 훔치러 가야겠군!",
+      4: "훌륭한 계획은 항상 방해를 받는다니깐.",
+      5: "이건 나빠… 나빠나빠나빠나빠나빠! 플라스마단에 나빠! 줄여서, 플라나빠!"
     },
   },
   "flare_grunt": {
     "encounter": {
-      1: "패션이 우리한텐 가장 중요하다고!"
+      1: "네 포켓몬, 플레어단의 우아함에 상대가 되지 않는다고.",
+      2: "선글라스를 가져왔길 바랄게, 곧 밝게 빛날 테니까!",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:469]:
```suggestion
      2: "선글라스를 가져왔길 바랄게. 곧 밝게 빛날 테니까!",
```

File: src/locales/ko/dialogue.ts [hunk 30]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      3: "플레어단이 세상의 불완전함을 정화할 거야!",
+      4: "플레어단의 광채를 마주칠 준비는 됐겠지!",
+      5: "패션이 우리한텐 가장 중요하다고!"
     },
     "victory": {
-      1: "미래가 밝아 보이질 않네."
+      1: "미래가 밝아 보이질 않네.",
+      2: "생각했던 거 보다 전투에는 많은 요소가 있는 것 같군. 다시 계획을 짜야겠어.",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:476]:
```suggestion
      2: "생각했던 것보다 전투에는 많은 요소가 있는 것 같군. 다시 계획을 짜야겠어.",
```

File: src/locales/ko/dialogue.ts [hunk 31]
```diff
@@ -385,50 +385,98 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      3: "아앗?! 내가 졌어?!",
+      4: "패배 속에서도, 플레어단의 우아함은 빛나고 있다고.",
+      5: "네가 이긴 것 같네, 그렇지만 졌어도, 난 우아하게 퇴장할 거니까!"
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:479]:
```suggestion
      5: "네가 이긴 것 같네. 그렇지만 졌어도, 난 우아하게 퇴장할 거니까!"
```

**Review by returntoice — COMMENTED** [review]
Some Korean grammar corrections

**Review by sodaMelon — APPROVED** [review]

File: src/locales/ko/dialogue.ts (hunk 30 above)
Comment (sodaMelon) [reply src/locales/ko/dialogue.ts:476]: thankyou!!!

**Comment by sodaMelon** [comment]
> Some Korean grammar corrections

I might have played too many games in English and Japanese lol thanks for the fix

**Comment by EnochG1** [comment]
@Blitz425 
From now Japanese is added in beta.
Could you merge latest beta and add Japanese locale files? 🙏

**Pull request merged**